app/webpack/observations/identify/components/observation_modal.jsx

```diff
@@ -738,6 +738,24 @@ class ObservationModal extends React.Component {
                                 I18n.t( `places_name.${_.snakeCase( country.name )}`, { defaultValue: country.name } ) || I18n.t( "somewhere_on_earth" )
                               ) : I18n.t( "somewhere_on_earth" ) }
                             </li>
+                            { observation.application && observation.application.name && (
+                              <li>
+                                <a href={observation.application.url}>
+                                  { DISPLAYABLE_APP_IDS.includes( observation.application.id ) ? (
```

Comment:
I think you should make this a prop (and name it something more descriptive like `officialAppIds`, since you're displaying something regardless of whether it's in this array) and set the default value to an empty array. You can still set a constant in the haml template like you're doing, but maybe store it in the `config` part of state in `webpack-entry.js` like we're doing with `PREFERRED_PLACE`, then pass it to this component as a prop from the container. The upshot of all that is that `ObservationModal` can be reused and not crash even when a developer fails to pass in this array of IDs, and a developer can supply the IDs in different ways in different contexts (maybe setting a global constant isn't possible in some situations).